src/components/NavBreadcrumbs.js

```diff
@@ -28,7 +28,7 @@ export default function NavBreadcrumb(props) {
     if (i.href !== null || i.href !== '') {
```

Comment:
null & empty string are both falsy values. I believe this if statement can be rewritten.
```suggestion
    if (i.href) {
```

Reply:
The NavBreadcrumb component was written by @shinjonathan. I do agree with your assessment though.